styles.js

```diff
@@ -0,0 +1,13 @@
+import { createGlobalStyle } from "styled-components";
+
+export default createGlobalStyle`
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+  body {
+    margin: 0;
+    font-family: system-ui;
+  }
+`;
```

Comment:
you need to add the GlobalStyles in the _app.js as well, otherwise they will not have an effect.

Reply:
import "../global.css";
import { SWRConfig } from "swr";
import GlobalStyle from "../styles";
export default function App({ Component, pageProps }) {
  return (
    <>
      <GlobalStyle />

      <SWRConfig
        value={{
          fetcher: (resource, init) =>
            fetch(resource, init).then((response) => response.json()),
        }}
      >
        <Component {...pageProps} />
      </SWRConfig>
    </>
  );
}